tests/e2e/flowtest/test_selinux_relabel_solution.py

```diff
@@ -0,0 +1,239 @@
+import logging
+import os.path
+
+import pytest
+import yaml
+from ocs_ci.ocs.resources import pod
+import time
+from tempfile import mkdtemp
+import os
+from ocs_ci.ocs.resources.pod import validate_pods_are_respinned_and_running_state
+from ocs_ci.framework.testlib import E2ETest, ignore_leftovers
+from ocs_ci.ocs import ocp, constants
+from ocs_ci.helpers import helpers
+import urllib.request
+from ocs_ci.ocs.resources.pod import delete_deploymentconfig_pods
+from ocs_ci.ocs.resources import pod as res_pod
+from ocs_ci.utility.utils import run_cmd
+from ocs_ci.utility.templating import dump_data_to_temp_yaml
+
+
+TARFILE = "file.gz"
+ntar_loc = mkdtemp()
+
+log = logging.getLogger(__name__)
+
+
+class TestSelinuxrelabel(E2ETest):
+    @pytest.fixture()
+    def download_files(self):
```

Comment:
There seems to be a perf image with this kernel file downloaded in it, should we use it, instead?